src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -179,6 +179,19 @@ if (__DEV__) {
           parent.tagName,
         );
     testElement.innerHTML = html;
+
+    if (parent.tagName === 'NOSCRIPT') {
+      // <noscript> content is parsed as text, but only if the browser parses it
+      // together with <noscript> tag itself. So we have to wrap it once more.
+      var wrapperElement = document.createElement('div');
+      wrapperElement.innerHTML = '<noscript>' + html + '</noscript>';
+      var noscriptElement = wrapperElement.firstElementChild;
```

Comment:
Why not just `firstChild` (since it goes back longer in browser support and seems sufficient)?